src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java

```diff
@@ -38,4 +46,20 @@ public static void printHelp() {
                 "Go back to the workout menu."));
         printer.showToUser(helpMessage.toString().trim());
     }
+
+    public static void emptyInputError() {
+        System.out.println("Please enter something.");
+    }
+
```

Comment:
Can try convert to throw Schwagserexceptions which can be handled by SchwasadsfgerHandler. That will help format the output which aligns with Ui output format